engine/src/main/java/org/terasology/world/internal/WorldProviderWrapper.java

```diff
@@ -44,26 +46,51 @@ public boolean isBlockRelevant(Vector3i pos) {
         return core.isBlockRelevant(pos.x, pos.y, pos.z);
     }
 
+    @Override
+    public boolean isBlockRelevant(Vector3ic pos) {
+        return core.isBlockRelevant(pos.x(), pos.y(), pos.z());
+    }
+
     @Override
     public boolean isBlockRelevant(Vector3f pos) {
         return isBlockRelevant(new Vector3i(pos, RoundingMode.HALF_UP));
     }
 
+    @Override
+    public boolean isBlockRelevant(Vector3fc pos) {
+        return isBlockRelevant(new org.joml.Vector3i(pos, org.joml.RoundingMode.HALF_UP));
```

Comment:
Explicit package like this makes me sad - can we avoid it via imports or is something in the way of that?

Reply:
nope, Vector3i conflicts with Vector3i from termath.